tests/app/celery/test_tasks.py

```diff
@@ -970,79 +889,28 @@ def test_should_not_save_sms_if_restricted_service_and_invalid_number(self, noti
         job = create_job(template)
         notification = _notification_json(template, to="07700 900849")
 
-        save_sms_mock = mocker.patch("app.celery.tasks.save_sms")
-
-        process_row(
-            Row(
-                {"foo": "bar", "to": notification["to"]},
-                index="row_num",
-                error_fn=lambda k, v: None,
-                recipient_column_headers=["to"],
-                placeholders={"foo"},
-                template=SMSMessageTemplate(template.__dict__),
-            ),
-            template,
-            job,
-            service,
-        )
-
-        assert not save_sms_mock.called
-
-    def test_should_not_save_sms_if_team_key_and_recipient_not_in_team(self, notify_db_session, mocker):
-        user = create_user(mobile_number="6502532222")
-        service = create_service(user=user, restricted=True)
-        template = create_template(service=service)
-        job = create_job(template)
-        notification = _notification_json(template, to="07700 900849")
-
-        team_members = [user.mobile_number for user in service.users]
-        assert "07890 300000" not in team_members
-
-        save_sms_mock = mocker.patch("app.celery.tasks.save_sms.apply_async")
-
-        process_row(
-            Row(
-                {"foo": "bar", "to": notification["to"]},
-                index="row_num",
-                error_fn=lambda k, v: None,
-                recipient_column_headers=["to"],
-                placeholders={"foo"},
-                template=SMSMessageTemplate(template.__dict__),
-            ),
+        save_sms_mock = mocker.patch("app.celery.tasks.save_smss")
+
+        process_rows(
+            [
+                Row(
+                    {"foo": "bar", "to": notification["to"]},
+                    index="row_num",
+                    error_fn=lambda k, v: None,
+                    recipient_column_headers=["to"],
+                    placeholders={"foo"},
+                    template=SMSMessageTemplate(template.__dict__),
+                )
+            ],
             template,
             job,
             service,
         )
 
         assert not save_sms_mock.called
 
-    def test_should_not_save_email_if_restricted_service_and_invalid_email_address(self, notify_db_session, mocker):
```

Comment:
Do we have another test to replace this one?

Reply:
same here: https://github.com/cds-snc/notification-api/pull/1502/files